src/content/en/updates/2017/02/improved-add-to-home-screen.md

```diff
@@ -180,20 +180,20 @@ questions you might have.
 - - -
 
 ## FAQs 
-### What are the requirements for a site to use improved add to Home screen?
+### What are the requirements for a site to use the improved add to Home screen?
 
 The requirements are designed to be the same as the [technical requirements for
-the add to home screen
+the add to Home screen
```

Comment:
The official Android practice is for home to be lower case. Please fix throughout.